pkgs/applications/networking/instant-messengers/cordless/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "cordless";
+  version = "2020-08-30";
+
+  src = fetchFromGitHub {
+    owner = "Bios-Marcel";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-CwOI7Ah4+sxD9We+Va5a6jYat5mjOeBk2EsOfwskz6k=";
+  };
+
+  subPackages = [ "." ];
+
+  vendorSha256 = "sha256-01I7GrZkaskuz20kVK2YwqvP7ViPMlQ3BFaoLHwgvOE=";
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/Bios-Marcel/cordless";
+    description = "The Discord terminal client you never knew you wanted.";
```

https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes
```suggestion
    description = "Discord terminal client";
```

---

haha, I thought about changing it. For the record that their GitHub tag line and not my editorializing :P. I'll switch it.